ground-station-openmct/terminal/terminal_logic.js

```diff
@@ -43,8 +43,9 @@ form.addEventListener("submit", function (event) {
       return response;
     })
     .then((response) => {
-        const responseText = response.ack !== undefined ? `${response.message}${response.ack}` : response.message;
-        terminal.innerHTML += `$ ${response.message}\n`;
+        if (response.ack !== undefined) {
```

Comment:
If you do response.ack != undefined, then any messages for clear or invalid command don't print. Just do response != undefined not response.ack != undefined since we still want to see the "invalid command" message